pulumi/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,10 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$bindir   = Join-Path -Path $toolsDir -ChildPath "bin"
+
+Uninstall-ChocolateyPath $bindir 'Machine'
+
+if (Test-Path $toolsDir) {
+    Remove-Item $toolsDir -Recurse -Force
+}
+
+refreshenv
```

Comment:
I believe this only works when users have the Chocolatey profile loaded. `Update-SessionEnvironment` is the powershell equivalent. 
```suggestion
Update-SessionEnvironment
```